footer.php

```diff
@@ -1,96 +1,123 @@
-        <footer class="footer">
-            <?php
-                if(!isset($_COOKIE['gdpr'])) {
-                    $args = Array(
-                        'category_name'  =>  'GDPR',
-                    );
-    
-                    $gdpr_post = get_posts($args);
-                } else {
-                    $gdpr_post = Array();
-                }
-            ?>
+<?php
+/**
+ * Footer
+ *
+ * Footer file for theme
+ *
+ * @package WordPress
+ * @subpackage community-portal
+ * @version 1.0.0
+ * @author  Playground Inc.
+ */
 
-            <?php if(sizeof($gdpr_post) > 0): ?>
-            <div class="gdpr">
-                <div class="gdpr__container">
-                    <a href="#" class="gdpr__close">
-                        <svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
-                            <path d="M17 1L1 17" stroke="#F9F9FA" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
-                            <path d="M1 1L17 17" stroke="#F9F9FA" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
-                        </svg>
-                    </a>
+?>
 
-                    <h2 class="gdpr__title"><?php print $gdpr_post[0]->post_title; ?></h2>
-                    <?php
-                        print  $gdpr_post[0]->post_content;
-                    ?>
-                    <div class="gdpr__cta-container">
-                        <a href="#" class="gdpr__cta"><?php _e('Accept', 'community-portal'); ?></a>
-                    </div>
-                </div>
-            </div>
-            <?php endif; ?>
-            <div class="footer__container">
-                <div class="footer__logo-container">
-                    <img src="<?php print get_stylesheet_directory_uri(); ?>/images/footer-logo.png"  class="footer__logo" alt="><?php _e('Mozilla Logo', 'community-portal'); ?>" />
-                </div>
-                <div class="footer__menu-container">
-                    <?php 
-                        wp_nav_menu(array(
-                                        'menu'           =>  'Footer Primary',
-                                        'menu_class'        =>  'footer-menu footer-menu--primary')
-                        ); 
-                    ?>
-                    
-                    <?php 
-                        wp_nav_menu(array(                                 
-                                        'menu'           =>  'Mozilla Main Menu',
-                                        'menu_class'        =>  'footer-nav-menu')
-                        ); 
-                    ?>
+		
```

Comment:
This blank line is throwing the following warning: Found precision alignment of 1 spaces.

Reply:
Fixed